tmt.spec

```diff
@@ -1,250 +1,109 @@
+Source:         %{url}/releases/download/%{version}/tmt-%{version}.tar.gz
 
-%description
-The tmt Python module and command line tool implement the test
-metadata specification (L1 and L2) and allows easy test execution.
-This package contains the command line tool.
-
-%?python_enable_dependency_generator
-
-
-%package -n     python%{python3_pkgversion}-%{name}
-Summary:        Python library for the %{summary}
-BuildRequires: python%{python3_pkgversion}-devel
-BuildRequires: python%{python3_pkgversion}-docutils
-BuildRequires: python%{python3_pkgversion}-setuptools
-BuildRequires: python%{python3_pkgversion}-pytest
-BuildRequires: python%{python3_pkgversion}-click
-BuildRequires: python%{python3_pkgversion}-fmf >= 1.2.0
-BuildRequires: python%{python3_pkgversion}-requests
-BuildRequires: python%{python3_pkgversion}-testcloud >= 0.9.2
-BuildRequires: python%{python3_pkgversion}-markdown
-BuildRequires: python%{python3_pkgversion}-junit_xml
-BuildRequires: python%{python3_pkgversion}-ruamel-yaml
-BuildRequires: python%{python3_pkgversion}-jinja2
-# Only needed for rhel-8 (it has python3.6)
-%if 0%{?rhel} == 8
-BuildRequires: python%{python3_pkgversion}-typing-extensions
-BuildRequires: python%{python3_pkgversion}-dataclasses
-BuildRequires: python%{python3_pkgversion}-importlib-metadata
-%endif
-# Required for tests
-BuildRequires: rsync
-%{?python_provide:%python_provide python%{python3_pkgversion}-%{name}}
+BuildArch:      noarch
+BuildRequires:  python3-devel
 
-%description -n python%{python3_pkgversion}-%{name}
+%global _description %{expand:
 The tmt Python module and command line tool implement the test
-metadata specification (L1 and L2) and allows easy test execution.
-This package contains the Python 3 module.
-
-%package provision-container
-Summary: Container provisioner for the Test Management Tool
-Obsoletes: tmt-container < 0.17
-Requires: tmt == %{version}-%{release}
-Requires: podman
-Requires: (ansible or ansible-collection-containers-podman)
-
-%description provision-container
-Dependencies required to run tests in a container environment.
-
-%package provision-virtual
-Summary: Virtual machine provisioner for the Test Management Tool
-Obsoletes: tmt-testcloud < 0.17
-Requires: tmt == %{version}-%{release}
-Requires: python%{python3_pkgversion}-testcloud >= 0.9.2
-Requires: libvirt-daemon-config-network
-Requires: openssh-clients
-Requires: (ansible or ansible-core)
+metadata specification (L1 and L2) and allows easy test execution.}
+
+%description %_description
+
+Summary:        %{summary}
+Requires:       git-core rsync sshpass
+
+Obsoletes:      tmt-all
+Obsoletes:      tmt-report-html
+Obsoletes:      tmt-report-junit
+Obsoletes:      tmt-report-polarion
+Obsoletes:      tmt-report-reportportal
+Obsoletes:      tmt-test-convert
+
+Recommends:     tmt+test-convert
+Recommends:     tmt+export-polarion
+Recommends:     tmt+provision
+Recommends:     tmt+report-junit
+Recommends:     tmt+report-polarion
```

Comment:
Careful, lines like this below `%description` have a tendency of becoming part of the description verbatim, rather than as the actual metadata of the package.